### DISCUSSION TITLE
Camplayer OS on Pi4 with POE

### DISCUSSION THREAD

**Discussion by jammapic** [raspicamplayer/camplayer#55] [Camplayer OS]
Anyone else seen the problem with the POE hat fan not running?

Thanks

**Comment by freddiebd** [comment]
Do you happen to still have a copy of the Camplayer OS image? Link on their website doesn't work anymore :(

**Reply by jammapic** [reply]
Sadly not. I needed one when I was rebuilding - but ended up having to re image from the old memory card. 

I can perhaps do you a straight image from my working memory card if that helps?

**Reply by freddiebd** [reply]
That would be insanely awesome! Do you want to do it via WeTransfer https://wetransfer.com/ and then share the transfer link with me?

**Reply by freddiebd** [reply]
Or if easier @jammapic , please email to freddiebd(at)proton(dot)me
